src/app/loans/loans-routing.module.ts

```diff
@@ -11,22 +11,26 @@ import { LoansViewComponent } from './loans-view/loans-view.component';
 import { GeneralTabComponent } from './loans-view/general-tab/general-tab.component';
 import { AccountDetailsComponent } from './loans-view/account-details/account-details.component';
 import { NotesTabComponent } from './loans-view/notes-tab/notes-tab.component';
+import { OverdueChargesTabComponent } from './loans-view/overdue-charges-tab/overdue-charges-tab.component';
+import { OriginalScheduleTabComponent } from './loans-view/original-schedule-tab/original-schedule-tab.component';
+import { RepaymentScheduleTabComponent } from './loans-view/repayment-schedule-tab/repayment-schedule-tab.component';
+import { TransactiosTabComponent } from './loans-view/transactios-tab/transactios-tab.component';
+import { AddCollateralComponent } from './add-collateral/add-collateral.component';
+import { ChargesTabComponent } from './loans-view/charges-tab/charges-tab.component';
 
 /** Custom Resolvers */
 import { LoanChargeTemplateResolver } from './common-resolvers/loan-charge-template.resolver';
 import { LoanDetailsResolver } from './common-resolvers/loan-details.resolver';
 import { LoanDetailsGeneralResolver } from './common-resolvers/loan-details-general.resolver';
 import { LoanNotesResolver } from './common-resolvers/loan-notes-resolver';
-import { ChargesTabComponent } from './loans-view/charges-tab/charges-tab.component';
 import { LoanDetailsChargesResolver } from './common-resolvers/loan-details-charges.resolver';
-import { OverdueChargesTabComponent } from './loans-view/overdue-charges-tab/overdue-charges-tab.component';
-import { OriginalScheduleTabComponent } from './loans-view/original-schedule-tab/original-schedule-tab.component';
-import { RepaymentScheduleTabComponent } from './loans-view/repayment-schedule-tab/repayment-schedule-tab.component';
-import { TransactiosTabComponent } from './loans-view/transactios-tab/transactios-tab.component';
+import { LoanCollateralTemplateResolver } from './common-resolvers/collateral-template.resolver';
 
+/** Loans Route. */
 const routes: Routes = [
   {
-    path: '',
+    path: 'loans',
```

Comment:
Please remove this change as well

Reply:
This was the mistake for getting that issue